cmake/adjust/flags.cmake

```diff
@@ -173,7 +213,7 @@ function(nbl_adjust_flags)
 			
 			set(MAPPED_CONFIG $<TARGET_GENEX_EVAL:${NBL_TARGET_ITEM},$<TARGET_PROPERTY:${NBL_TARGET_ITEM},NBL_CONFIGURATION_MAP>>)
 			
-			if(MSVC)
+			if(MSVC AND CMAKE_CXX_COMPILER_ID STREQUAL MSVC)
```

Comment:
`if(CMAKE_CXX_COMPILER_ID STREQUAL MSVC)`

Reply:
ah was pretty sure i changed this..